### PR TITLE
Remove deprecated scripts and BLAS list files

### PR DESCRIPTION
**Pull request by vmalia**
As a result of https://github.com/uxlfoundation/oneMath/discussions/598 discussion, this PR removes the outdated scripts and updates the `create_new_backend` document accordingly.

Document preview: https://github.com/uxlfoundation/oneMath/blob/96e0d13ee2224619af396b892a58926236e36476/docs/create_new_backend.rst

### CODE REVIEW

File: scripts/generate_cmake.py [hunk 1]
```diff
@@ -1,129 +0,0 @@
-#!/usr/bin/env python
```
Comment (Rbiessy) [inline scripts/generate_cmake.py:1]: We should also remove this line: https://github.com/uxlfoundation/oneMath/blob/develop/.github/CODEOWNERS#L14

**Review by Rbiessy — COMMENTED** [review]
LGTM, overall!

File: scripts/generate_cmake.py (hunk 1 above)
Comment (vmalia) [reply scripts/generate_cmake.py:1]: Good catch. Updated.

**Review by Rbiessy — APPROVED** [review]

**Review by spencerpatty — APPROVED** [review]
changes look fine to me.  nice to see the "+" and "-" highlighted in the examples ... makes it simpler to follow what is necessary.   
 The text changes and removal of python scripts is also good, to focus on most important pieces of changes needed.  Approved for merge.

**Pull request merged**